api/v1alpha1/oadp_types.go

```diff
@@ -160,6 +160,12 @@ type NodeAgentConfig struct {
 	// +kubebuilder:validation:Enum=restic;kopia
 	// +kubebuilder:validation:Required
 	UploaderType string `json:"uploaderType"`
+
+	// UnsupportedNodeAgentArgs accepts a string of server args and their values to be supplied to nodeagent daemon set
+	// each pair of server arg and value is comma separated if there are multiple args to be supplied
+	// This will replace and override any other args specified via DPA for the nodeagent
+	// +optional
+	UnsupportedNodeAgentArgs string `json:"unsupportedNodeAgentArgs,omitempty"`
```

Comment:
the name is a little confusing to me: since we adding a way to be used by OADP, should the name not contain Unsupported?

Reply:
the use of the term "unsupported" was done at my request.  Due to the fact a customer could potentially apply any key/value pair the use of this spec field must indicate that it's unsupported by default.

As customers have a need to inject a config option that we do not yet have in the DPA officially we will support them.

Long term this allows the OADP to help customers in situations where we find a new or previously ignored config option.  Graduating config directly to the DPA would be a good workflow imho.

Reply:
Maybe we say `OverrideNodeAgentArgs` ? I am fine ether way.

Reply:
no.. I still want unsupported to be in there at this time.